test/travis_test/build_2_container.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+BUILD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+code_dir=$(dirname $BUILD)
+code_dir=$(dirname $code_dir)
+echo "build path is $BUILD"
+echo "code path is $code_dir"
+
+# Initialize the needed mizar submodule
+# git submodule update --init --recursive
+
+# Create and Start the build contrainer
+echo "    --- images_1 ---"
+docker images
+docker build -f $code_dir/build/Dockerfile -t aca_build0 .
+echo "    --- images_2 ---"
+docker images
+echo "    --- container-1 ---"
+docker ps -a
+docker rm -f aca_PARENT || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_PARENT aca_build0:latest /bin/bash
+docker start aca_PARENT
+
+docker rm -f aca_CHILD || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_CHILD aca_build0:latest /bin/bash
+docker start aca_CHILD
+echo "    --- container-2 ---"
+docker ps -a
+# Build mizar first
```

Comment:
this is looking good, let's do some clean up and finalize this. Please go ahead and remove line 29-30.